pkg/scheduler/scheduler_test.go

```diff
@@ -480,6 +480,7 @@ func TestSchedulerMultipleProfilesScheduling(t *testing.T) {
 
 	// Run scheduler.
 	informerFactory.Start(ctx.Done())
+	informerFactory.WaitForCacheSync(ctx.Done())
```

Comment:
To avoid similar flakes, can we add a utility function in UT like what we did in the integration test? And then turn all UTs' logic of starting informerFactory to call the utility.

https://github.com/kubernetes/kubernetes/blob/04362870ad9f9812f487ab8f89a40f6bf2daa588/test/integration/util/util.go#L184-L188

Reply:
mind doing it in a followup?